header-home.php

```diff
@@ -37,6 +37,7 @@
 </head>
 
 <body <?php body_class(); ?>>
+	<div id="skip"><a href="#content">Skip to Main Content</a></div>
```

Comment:
there's no #content div on the homepage so can we have this one skip to #search-main?